### PR TITLE
[v10] Warn user when desktop is active (#1297)

### PR DESCRIPTION
**Pull request by ibeckermayer**
Backports #1297 to v10

I found a stumper of a webpack dev server error when manually testing which was fixed by b27c1b831ac2a403519aef2c640899317d21b930

Do not merge until https://github.com/gravitational/teleport/pull/17936 is merged or desktop sessions will break.

### CODE REVIEW

File: packages/teleport/src/Apps/AddApp/AddApp.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ import * as Icons from 'design/Icon';
 
 import useTeleport from 'teleport/useTeleport';
 
-import { TabIcon } from 'teleport/components/Tabs';
+import TabIcon from 'teleport/components/Tabs/TabIcon';
```
Comment (ravicious) [inline packages/teleport/src/Apps/AddApp/AddApp.tsx:25]: Importing the named export doesn't appear to be an issue on my machine.

But [I've run into a similar problem](https://gravitational.slack.com/archives/C02DQ1C2BMW/p1667223568113269) with webpack today. Could you check if removing the `packages/teleport/node_modules/.cache` directory helps?

**Review by ravicious — APPROVED** [review]
Leaving an approval but please check if that problem with the import was an issue with cache. If so, we should use the named export.

**Review by hatched — APPROVED** [review]
Approved with confirmation of @ravicious's comment.

File: packages/teleport/src/Apps/AddApp/AddApp.tsx (hunk 1 above)
Comment (ibeckermayer) [reply packages/teleport/src/Apps/AddApp/AddApp.tsx]: Yeah that fixed it.

**Pull request merged**